lib/doorkeeper/models/concerns/scopes.rb

```diff
@@ -7,6 +7,14 @@ def scopes
         OAuth::Scopes.from_string(scopes_string)
       end
 
+      def scopes=(value)
+        if value.is_a? Array
+          super value.join(' ')
```

Comment:
Minor suggestion, these 4 lines could be replaced with `super Array(value).join(' ')`